microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/hover/PropertiesHoverParticipant.java

```diff
@@ -126,6 +140,9 @@ public Hover collectHover(JavaHoverContext context, IProgressMonitor monitor) th
 			return null;
 		}
 
+		// property references may be surrounded by curly braces in Java file

```

Comment:
Is it legal for property names to contain curly braces ? If not, this should be pretty safe to do in all cases. Otherwise we can polish this up.

Reply:
In the case of `@ConfigProperty`, you could (if you wanted to) have something like `name = {greeting.message}` where in application.properties `{greeting.message}=hello` . In fact it doesn't work exactly since hovering over the property in the Java file says the property is not set, but in the application file there is no validation error for it.

So I think we should be fine treating curly braces as special in `@Scheduled` member values since we really do need them as the value can be a literal or a property reference.